pkg/instance/docs.go

```diff
@@ -64,6 +64,16 @@ type _ struct {
 	ID uint `json:"id"`
 }
 
+// swagger:parameters deleteDeploymentInstance
+type _ struct {
+	// in: path
+	// required: true
+	ID uint `json:"id"`
+	// in: path
+	// required: true
+	InstanceID uint `json:"instanceId"`
```

Comment:
At some point we were moving to accepting names instead of ids. Does this also make sense here? Users deleting an instance via its name?

Reply:
Yeah, but that was due to an endpoint translating the name to its id. Identifying stacks and groups by name brought some issues in regards to how Gorm is used... In other words, let's keep this simple for now :smile: